src/arproc.py

```diff
@@ -1079,22 +1078,26 @@ def rn_frame(slf, det):
     Returns
     -------
     rn_img : ndarray
-
+      Read noise *variance* image (i.e. RN**2)
     """
     # Loop on amplifiers
     rnimg = np.zeros_like(slf._ampsec[det-1])
     for ii in range(slf._spect['det'][det-1]['numamplifiers']):
         amp = ii+1
         amppix = slf._ampsec[det-1] == amp
-        rnimg[amppix] = (slf._spect['det'][det-1]['ronoise'][ii]**2 +
-                         (0.5*slf._spect['det'][det-1]['gain'][ii])**2)
+        #rnimg[amppix] = (slf._spect['det'][det-1]['ronoise'][ii]**2 +
+        #                 (0.5*slf._spect['det'][det-1]['gain'][ii])**2)
+        # Assumes readnoise is given in e- not counts
+        rnimg[amppix] = slf._spect['det'][det-1]['ronoise'][ii]**2
```

Comment:
Why has the gain term been removed?  This term accounts for the uncertainty in the rounding error associated with the readout. I can't see where this term is accounted for elsewhere in the code.